numpy/distutils/core.py

```diff
@@ -18,8 +18,6 @@
 import warnings
 import distutils.core
 import distutils.dist
-
-from numpy.distutils.extension import Extension
```

Comment:
This breaks code that expect to be able to import `Extension` from here.

Reply:
I am sorry for bring total noob because this is my first contribution but can you please tell me that do i need to commit again with the for loop variable issue and import extension again to fix the crash or just make a commit of import to fix the crash.
And if there is something i need to do .... Please do tell.
Thanks